crates/pop-parachains/src/new_parachain.rs

```diff
@@ -19,22 +22,22 @@ use walkdir::WalkDir;
 /// * `tag_version` - version to use (`None` to use latest).
 /// * `config` - customization values to include in the new parachain.
 pub fn instantiate_template_dir(
-	template: &Template,
+	template: &ParachainTemplate,
 	target: &Path,
 	tag_version: Option<String>,
 	config: Config,
 ) -> Result<Option<String>> {
 	sanitize(target)?;
 
-	if template.matches(&Provider::Pop) {
+	if Provider::Pop.matches(&template) {
```

Comment:
```suggestion
	if Provider::Pop.provides(&template) {
```

Might be a better name, or `supports`